README.md

```diff
@@ -114,9 +136,13 @@ $ dbt-coverage compare after.json before.json  # Generate a detailed coverage de
 
 ## Related packages
 
-https://github.com/mikaelene/dbt-test-coverage
+- https://github.com/mikaelene/dbt-test-coverage
+- [interrogate](https://interrogate.readthedocs.io/en/latest/) (docs coverage for Python)
+- [coverage.py](https://github.com/nedbat/coveragepy) (execution coverage for Python)
 
 ## License
 
 Licensed under the MIT license (see [LICENSE.md](LICENSE.md) file for more
 details).
+
+[![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2Fslidoapp%2Fdbt-coverage.svg?type=large)](https://app.fossa.com/projects/git%2Bgithub.com%2Fslidoapp%2Fdbt-coverage?ref=badge_large)
```

Comment:
This badge is really nice, I'm just thinking whether we shouldn't add some heading above. Maybe Fossa?

```suggestion
## Fossa

[![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2Fslidoapp%2Fdbt-coverage.svg?type=large)](https://app.fossa.com/projects/git%2Bgithub.com%2Fslidoapp%2Fdbt-coverage?ref=badge_large)
```

Reply:
@sweco I don't think it needs that -- the link should be enough 🙂

Reply:
I just think that now it seems like a part of the License. Or you don't think it's an issue?

![image](https://user-images.githubusercontent.com/11132999/145080874-a78ac360-fad1-4890-adf1-aaff37b9b1bb.png)

Reply:
@sweco well, some use it as a full replacement of the License part, so in that sense, I don't find it to be an issue.

https://github.com/codecov/python-standard/blob/master/README.md#license

That said, I don't mind adding a new header, I just didn't find it to be necessary 🙂

Reply:
> some use it as a full replacement of the License part

Ah, I see! 😊  Thanks for the explanation, no need to add it.